src/sql/src/plan/expr.rs

```diff
@@ -207,15 +207,19 @@ impl WindowExpr {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
 /// A window function with its parameters.
 ///
-/// There are two types of window functions: scalar window functions, that
-/// return a different scalar value for each row within a partition, and
-/// aggregate window functions, that return the same value for all the tuples
-/// within the same partition. Aggregate window functions can be computed
-/// by joining the input relation with a reduction over the same relation
-/// that computes the aggregation using the partition key as its grouping
-/// key.
+/// There are three types of window functions: scalar window functions, that
```

Comment:
can you break this definition of the three window functions into a bulleted list or table?

Reply:
Done! Kept the text the same, though: happy to hear suggestions on how to improve it, as it was one of the last things I did before I opened my PR, and my brain was already taking a break at that time.